node-launchpad/src/components/footer.rs

```diff
@@ -6,50 +6,126 @@
+                self.nodes_exist = nodes_exist;
+            }
+            _ => {}
         }
         Ok(None)
     }
 
     fn draw(&mut self, f: &mut crate::tui::Frame<'_>, area: Rect) -> Result<()> {
         let layer_zero = Layout::new(
             Direction::Vertical,
-            [Constraint::Min(1), Constraint::Max(3)],
+            [
+                // for the rest of the home scene
+                Constraint::Min(1),
+                // our footer
+                Constraint::Max(5),
+            ],
         )
         .split(area);
+        let border = Paragraph::new("").block(
+            Block::default()
+                .title("Key Commands")
+                .borders(Borders::ALL)
+                .border_style(Style::default().fg(COOL_GREY)),
+        );
+
+        let layer_one = Layout::new(
+            Direction::Vertical,
+            [
+                // border
+                Constraint::Length(1),
+                // line1
+                Constraint::Length(2),
+                // line2
+                Constraint::Length(1),
+                // border
+                Constraint::Length(1),
+            ],
+        )
+        .split(layer_zero[1]);
+
+        let text_style = if self.nodes_exist {
+            Style::default().fg(EUCALYPTUS)
+        } else {
+            Style::default().fg(LIGHT_PERIWINKLE)
+        };
 
-        let text: Cow<_> = match self.current_scene {
+        let command_style = if self.nodes_exist {
+            Style::default().fg(GHOST_WHITE)
+        } else {
+            Style::default().fg(LIGHT_PERIWINKLE)
+        };
+
+        let (line1, line2) = match self.current_scene {
             Scene::Home => {
-                "[Ctrl+g] Start nodes, [Ctrl+x] Stop nodes, [O] Set Resources, [D]iscord Username, [Q]uit".into()
+                let line1 = Line::from(vec![
+                    Span::styled(" [Ctrl+G] ", command_style),
+                    Span::styled("Start all Nodes       ", text_style),
+                    Span::styled("[Ctrl+X] ", command_style),
+                    Span::styled("Stop all Nodes         ", text_style),
+                    Span::styled("[H] ", Style::default().fg(GHOST_WHITE)),
+                    Span::styled("Help", Style::default().fg(EUCALYPTUS)),
+                ]);
+
+                let line2 = Line::from(vec![
+                    Span::styled(" [O] ", Style::default().fg(GHOST_WHITE)),
+                    Span::styled(
+                        "Manage Nodes               ",
+                        Style::default().fg(EUCALYPTUS),
+                    ),
+                    Span::styled("[D] ", Style::default().fg(GHOST_WHITE)),
+                    Span::styled(
+                        "Beta Rewards Programme      ",
+                        Style::default().fg(VERY_LIGHT_AZURE),
+                    ),
+                    Span::styled("[Q] ", Style::default().fg(GHOST_WHITE)),
```

Comment:
Maybe use lower case `o`, `d` and `q` here. Might be just me being pedantic, but I pressed `Shift` a few times.

Just looked at `dua`, which is quite a popular app using `ratatui`: https://github.com/Byron/dua-cli/blob/ced3b4f5e375278dbee52319eac8750b14eb328a/src/interactive/widgets/help.rs#L43

Reply:
The design is based of off a desgin doc shared on Slack, probably Jim is the guy you'd want to discuss this with.